src/MMLib.SwaggerForOcelot/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -78,8 +78,10 @@ private static void AddGatewayItSelfDocs(SwaggerGenOptions c, OcelotSwaggerGenOp
                     Title = "Gateway",
                     Version = OcelotSwaggerGenOptions.GatewayKey,
                 });
+              
+                AddDocumentFilterForGatewayItSelf(options.OcelotGatewayItSelfSwaggerGenOptions.DocumentFilterActions, c);
 
-                IncludeXmlComments(options.FilePathsForXmlCommentsOfGatewayItSelf, c);
+                IncludeXmlComments(options.OcelotGatewayItSelfSwaggerGenOptions.FilePathsForXmlComments, c);
```

Comment:
`OcelotGatewayItSelfSwaggerGenOptions` can be `null`. 
Am I right?

Reply:
Yes, you are right! It can be null, so I added null check.